extensions/notebook/src/test/book/book.test.ts

```diff
@@ -338,13 +419,13 @@ describe('BookTreeViewProviderTests', function () {
 
 		it('should show error message if config.yml file not found', async () => {
 			await bookTreeViewProvider.currentBook.readBooks();
-			should(bookTreeViewProvider.currentBook.errorMessage.toLocaleLowerCase()).equal(('Failed to read book '+ bookTreeViewProvider.currentBook.bookPath +': ENOENT: no such file or directory, open \'' + configFile + '\'').toLocaleLowerCase());
+			should(bookTreeViewProvider.currentBook.errorMessage.toLocaleLowerCase()).equal(('Failed to read book ' + bookTreeViewProvider.currentBook.bookPath + ': ENOENT: no such file or directory, open \'' + configFile + '\'').toLocaleLowerCase());
```

Comment:
Why are you lower casing this? And just use the already defined constant - saves having to update this if we decide to modify the text later https://github.com/microsoft/azuredatastudio/blob/main/extensions/notebook/src/common/localizedConstants.ts#L36